### DISCUSSION TITLE
how to log to a file with rotation?

### DISCUSSION THREAD

**Discussion by zhuravkov** [loco-rs/loco#1090] [Q&A] [answered]
Is it possible to set up logging to a file with rotation 1 time per day at the application level?

**Comment by kaplanelad** [comment, marked as answer]
Hey @zhuravkov
In the configuration YAML, you should use the [file_appender](https://github.com/loco-rs/loco/blob/master/src/config.rs#L119C9-L119C22).

Within the file appender configuration, there is a field called [rotation](https://github.com/loco-rs/loco/blob/master/src/config.rs#L142C9-L142C17). The value for this field should be set to never.

You can find all available options [here](https://github.com/loco-rs/loco/blob/master/src/logger.rs#L52).
